docs/source/classes/Jetscape/index.rst

```diff
@@ -6,20 +6,8 @@ Jetscape
 .. currentmodule:: Jetscape
 .. autoclass:: Jetscape
 
-.. automethod:: Jetscape.particle_list
+.. automethod:: Jetscape.create_loader
 .. automethod:: Jetscape.particle_objects_list
-.. automethod:: Jetscape.num_events
-.. automethod:: Jetscape.num_output_per_event
-.. automethod:: Jetscape.get_sigmaGen
-.. automethod:: Jetscape.particle_species
-.. automethod:: Jetscape.remove_particle_species
-.. automethod:: Jetscape.lower_event_energy_cut
-.. automethod:: Jetscape.charged_particles
-.. automethod:: Jetscape.uncharged_particles
-.. automethod:: Jetscape.strange_particles
-.. automethod:: Jetscape.pt_cut
-.. automethod:: Jetscape.rapidity_cut
-.. automethod:: Jetscape.pseudorapidity_cut
-.. automethod:: Jetscape.multiplicity_cut
-.. automethod:: Jetscape.particle_status
+.. automethod:: Jetscape._particle_as_list
```

Comment:
Why do you add a private method starting with an underscore to the documentation? Btw shouldn't it be two underscores?

Reply:
We can not use the double underscore in our case anymore, because we need to use that method in the loader now. We still mark it as private with one underscore, though. If we want it in the documentation or not we should discuss probably.

Reply:
Then just remove it from documentation